config/maps.txt

```diff
@@ -16,33 +16,40 @@ endmap
 # Production-level maps.
 
 map birdshot
-	#default
-	maxplayers 66
 	votable
+	minplayers 0
+	maxplayers 40
 endmap
 
 map deltastation
 	votable
+	minplayers 60
+	maxplayers 0
 endmap
 
 map icebox
 	votable
+	minplayers 40
+	maxplayers 0
 endmap
 
 map metastation
-	minplayers 25
-	#voteweight 0.5
+	default
 	votable
+	minplayers 0
+	maxplayers 80
```

Comment:
why does meta have an 80 playercap? it's equal with the other main stations.

Reply:
Meta feels very small. I could be wrong.